components/net/bluetooth_thread.rs

```diff
@@ -48,6 +52,48 @@ macro_rules! return_if_cached(
     );
 );
 
+macro_rules! check_blacklist {
```

Comment:
We should rename this. The current name doesn't indicate that it can return (which is the whole purpose of the macro I suppose).